signingscript/docker.d/build_msix_packaging.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set +e
+
+git clone https://github.com/mozilla/msix-packaging msix-packaging
+
+cd msix-packaging
+git checkout johnmcpms/signing
```

Comment:
Ideally, this gets replaced by a mainlined version of this branch, in Microsoft's repository. Whether or not that happens is TBD.

Reply:
This PR has been updated to point at Mozilla's repo; no movement on the Microsoft side.